### DISCUSSION TITLE
Grouping classes based on media screen

### DISCUSSION THREAD

**Discussion by patriicke** [tailwindlabs/tailwindcss#9407] [Ideas]
Why do we always need to write screen size on each class we want to include on the component?
I think we can include class grouping in classes with the same media screens.
Check out these.
![Screenshot from 2022-09-24 11-00-53](https://user-images.githubusercontent.com/97700182/192089735-1c5289b0-60ac-4acf-aafd-e657e114d83d.png)

DRY code principle and easy to read!
![Screenshot from 2022-09-24 10-47-10](https://user-images.githubusercontent.com/97700182/192089628-3d92fd41-3277-41b3-b95d-43744c95080d.png)

**Comment by wongjn** [comment]
[Here's a response to a similar suggestion](https://github.com/tailwindlabs/tailwindcss/discussions/9053#discussioncomment-3350470):

> We get this question so many times, so I would like to redirect you here for more information: https://twitter.com/adamwathan/status/1461519820411789314
